packages/analytics-js-integrations/src/integrations/ActiveCampaign/browser.js

```diff
@@ -19,22 +24,20 @@ class ActiveCampaign {
   }
 
   init() {
-    logger.debug('===In init ActiveCampaign===');
-
     if (!this.actId) {
-      logger.debug('actId missing');
+      logger.error(`${DISPLAY_NAME} : actId is required`);
```

Comment:
Since, you've already set the logger context above, the display name is not required.

Reply:
Correct this in all the integrations.

Reply:
updated